media/themes/Lunar/lunar.msstyles/lang/ru-RU.rc

```diff
@@ -0,0 +1,38 @@
+LANGUAGE LANG_RUSSIAN, SUBLANG_DEFAULT
+
+STRINGTABLE
+{
+1000, "ReactOS"

```

Comment:
Add 4 extra whitespaces here and in all other lines inside the brackets, to make the strings more readable. Although since it isn't done in other translations, then maybe do it for all languages in a separate PR.

Reply:
Not required.